dataland-frontend/src/components/resources/frameworkDataSearch/p2p/P2pDataModel.ts

```diff
@@ -238,8 +238,8 @@ export const p2pDataModel = [
             name: "absoluteEmissions",
             label: "Absolute emissions",
             description:
-              "Transparent accounting & disclosure of all greenhouse gas emissions (Scope 1, Scope 2, Scope 3) along the following metric: absolute emissions (t CO2e)",
-            unit: "",
+              "Transparent accounting & disclosure of all greenhouse gas emissions (Scope 1, Scope 2, Scope 3) along the following metric: absolute emissions",
```

Comment:
Please readd the units to the tooltip. They can be removed again once the units are displayed directly on the view panel (after switching to the new multilayer datatable)

Reply:
just this one or all instances?

Reply:
ok